lib/cc/services/github_pull_requests.rb

```diff
@@ -22,6 +30,37 @@ class Config < CC::Service::Config
 
   private
 
+  def report_status?
+    if should_apply_rollout?
+      rollout_allowed_by_username? || rollout_allowed_by_percentage?
+    else
+      true
+    end
+  end
+
+  def should_apply_rollout?
+    (github_login.present? || github_user_id.present?) &&
+      (config.rollout_usernames.present? || config.rollout_percentage.present?)
+  end
+
+  def rollout_allowed_by_username?
+    github_login.present? && config.rollout_usernames.present? &&
+      config.rollout_usernames.split(",").map(&:strip).include?(github_login)
+  end
+
+  def rollout_allowed_by_percentage?
+    github_user_id.present? && config.rollout_percentage.present? &&
+      github_user_id % 100 <= config.rollout_percentage
```

Comment:
Just confirmed it is an `Integer` in the db. I'd also assume that's what you'll get here too.